src/System.Runtime.Extensions/src/System/AppDomain.cs

```diff
@@ -205,10 +205,38 @@ public static bool MonitoringIsEnabled
         public event ResolveEventHandler ResourceResolve { add { } remove { } }
         //public void SetPrincipalPolicy(System.Security.Principal.PrincipalPolicy policy) { }
         /*
-        public void SetThreadPrincipal(System.Security.Principal.IPrincipal principal) 
-        { 
+        public void SetThreadPrincipal(System.Security.Principal.IPrincipal principal)
+        {
             if (principal == null) throw new ArgumentNullException("principal");
-            throw new PlatformNotSupportedException(); 
+            throw new PlatformNotSupportedException();
```

Comment:
just a stylistic change [removed trailing whitespace]  not related to my change, will revert it if problematic